src/npm.rs

```diff
@@ -18,10 +18,11 @@ pub fn npm_pack(path: &str) -> Result<(), Error> {
 }
 
 /// Run the `npm publish` command.
-pub fn npm_publish(path: &str) -> Result<(), Error> {
+pub fn npm_publish(path: &str, access: &str) -> Result<(), Error> {
```

Comment:
And then this argument would be `access: Access` and we know that we can't ever pass a bad access here